src/config.js

```diff
@@ -1,5 +1,8 @@
 export default {
   /* eslint-disable */
   githubSrc: 'https://raw.githubusercontent.com/ReproNim/reproschema/master/protocols/VoicePilot/VoicePilot_schema',
-  contentSrc: 'https://raw.githubusercontent.com/ReproNim/reproschema/master/protocols/VoicePilot/README.md'
+  contentSrc: 'https://raw.githubusercontent.com/ReproNim/reproschema/master/protocols/VoicePilot/README.md',
```

Comment:
is contentSrc used somewhere, since we have landingpage in the protocol now?

Reply:
if in case a protocol doesn't have landingPage, we can perhaps use contentSrc to point to some generic landing page. What say?

Reply:
i wouldn't do this. the protocol must have a landing page.

Reply:
if it doesn't the landing page will be empty and only show the button.

Reply:
ok, will update.